packages/swapper/src/api.ts

```diff
@@ -129,6 +129,10 @@ export type ApproveInfiniteInput<C extends ChainId> = {
   wallet: HDWallet
 }
 
+export type ApproveAmountInput<C extends ChainId> = ApproveInfiniteInput<C> & {
```

Comment:
Rename `ApproveInfiniteInput` to `ApproveInput` and make `ApproveInfiniteInput` an alias of the latter?